Entities/Attachable.cpp

```diff
@@ -331,7 +331,8 @@ namespace RTE {
 					Matrix atomRotationForSubgroup(rootParentAsMOSR->FacingAngle(GetRotAngle()) - rootParentAsMOSR->FacingAngle(rootParentAsMOSR->GetRotAngle()));
 					Vector atomOffsetForSubgroup = g_SceneMan.ShortestDistance(rootParentAsMOSR->GetPos(), m_Pos, g_SceneMan.SceneWrapsX());
```

Comment:
Can you consolidate this line into the one above - move the `.FlipX` into the same line as the ShortestDistance and clean up the TODO.